openstack_controller/datadog_checks/openstack_controller/api/api_rest.py

```diff
@@ -381,11 +381,29 @@ def use_legacy_nodes_resource(microversion):
         return self.make_paginated_request(url, 'nodes', 'uuid', params=params)
 
     def get_baremetal_conductors(self):
-        response = self.http.get(
-            '{}/v1/conductors'.format(self._catalog.get_endpoint_by_type(Component.Types.BAREMETAL.value))
-        )
-        response.raise_for_status()
-        return response.json().get('conductors', [])
+        def use_legacy_conductors_resource(microversion):
```

Comment:
This function is unused and can be removed

Reply:
Done.